src/main/thrift/octopus.thrift

```diff
@@ -0,0 +1,118 @@
+namespace scala com.gu.flexibleoctopus.model.thrift
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+/** 
+* A single story bundle that maps directly to a piece of content in Composer 
+*/
+struct StoryBundle {
+
+    1: required i64 id;
+
+    2: required string composerId;
+
+    3: required string pubCode;
+
+    4: required string sectionCode;
+
+    5: required Article body;
```

Comment:
Should perhaps be `bodyText` to match the enum and should also be optional as not every story bundle will contain an article of type `BODYTEXT`.